components/DropdownLight/DropdownLight.unit.test.jsx

```diff
@@ -19,31 +19,32 @@ describe('<DropdownLight />', () => {
   it('should return a value from item, using a object items', () => {
     render(<DropdownLight items={itemsObjectMock} />);
 
-    const dropdown = screen.getByRole('combobox');
-    userEvent.selectOptions(dropdown, 'Lemon');
+    const dropdown = screen.getByRole('button');
+    userEvent.click(dropdown);
 
-    const lemonItem = itemsObjectMock[0].value.toString();
+    const optionItem = screen.getByText('Lemon');
+    userEvent.click(optionItem);
 
-    expect(dropdown.value).toEqual(lemonItem);
+    const input = Number(screen.getByLabelText('selecione uma opcao').value);
```

Comment:
the content of value, in a form, generally is a String.
how about of change the value of object, in Mock, to a string?
With this, will not be necessary to convert the content to a Number.